local-dev/git/Dockerfile

```
@@ -1,12 +1,23 @@
-FROM unixtastic/git-ssh-server
+FROM alpine:3.10
```

Can we get this to 3.11 - the other images are mostly running on 3.11 now, alternatively, we could extend on amazeeio/commons to remove the need to manually increment?

```suggestion
FROM alpine:3.11
```

---

Good point, will bump the version.

---

version bumped